web/rootfs/etc/cont-init.d/10-config

```diff
@@ -72,6 +72,12 @@ if [[ ! -f /config/config.js ]]; then
             -e "s#// authdomain:.*#authdomain: '${XMPP_DOMAIN}',#" \
             /config/config.js
     fi
+
+    if [[ ! -z "${ETHERPAD_URL_BASE}" && -z "$(grep -om1 'etherpad_base:' /config/config.js)" ]]; then
+        sed -i \
+            -e "/enableWelcomePage/a\    etherpad_base: '${PUBLIC_URL}/etherpad/p/'," \
```

Comment:
have you tested if setting the etherpad_base option to just `/etherpad/p/` works ok? I think we shouldn't need the full URL there, and this way we can avoid having the PUBLIC_URL thing altogether.

Reply:
You are right. It works as absolute path instead url. Was changed in 253e638

Reply:
Fantastic! The we can also remove the PUBLIC_URL variable from this PR, since it's no longer needed here, right? (we need it for the transcriber settings, right?)

Reply:
Yes, you are right. Commit b003fc9 will remove it.